crates/rust-analyzer/src/to_proto.rs

```diff
@@ -1174,6 +1174,15 @@ pub(crate) fn runnable(
     })
 }
 
+pub(crate) fn test_info_2(
```

Comment:
You are totally correct, this is existed because at the very first, I am not sure what info the front-end might need.

Now I could say it only needs runables.